sklearn/model_selection/_search.py

```diff
@@ -1,6 +1,8 @@
 """
 The :mod:`sklearn.model_selection._search` includes utilities to fine-tune the
 parameters of an estimator.
+
+.. versionadded:: 0.18
```

Comment:
Maybe do a "versionchanged: 0.18" and as a change "moved from skearn.grid_search" or someting?